pkg/monitor/controller.go

```diff
@@ -365,3 +379,14 @@ func (monitor *controllerMonitor) deleteAgentCRD(name string) error {
 	}
 	return nil
 }
+
+func (monitor *controllerMonitor) AgentInfoCRDInstalled() bool {
+	_, err := monitor.apiExtensionClient.ApiextensionsV1().CustomResourceDefinitions().Get(context.TODO(), antreaAgentInfoCRDName, metav1.GetOptions{})
```

Comment:
Why this check is needed? I suppose CRD should be always there? Will CRD be deleted in any case?

Reply:
When antrea is running with agentless mode, CRD of antreaagentinfo may not create as it is not needed.